tsconfig.with-tests.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
What do you say, if we keep all configuration on the same `tsconfig.json`. We can exclude the files in the build with `.npmignore` or `.dockerignore`

Reply:
That could work, but there may be instances where we want the test files to end up in a container (for example) for testing environment purposes. I chose this solution because it addresses the problem at it's root: typescript. If we push the problem lower down, we need to maintain multiple config files.

Reply:
I'd strongly rather not have two tsconfig.json files. It's very easy for them to fall out of sync.

Is there any issue with tests landing in dist, or is it just a cosmetic thing?

Reply:
This was the best recommendation I found after researching it.

Tests don't belong in the built container - they add bulk and could potentially open security issues.

Reply:
If they don't belong in the container, why not `.dockerignore` and `.npmignore` them as Walter suggested?

I get the feeling that, if it's only a container issue, it should be handled by the container tooling...

Reply:
As I mentioned, sometimes we do want tests in the container. By 'built container', I mean production container.

I agree this is not the most elegant solution by having two configs, but I believe it is the quickest way to move forward.

Reply:
Makes sense. Let’s keep it as is for now then